src/Surfnet/AzureMfa/Domain/UserId.php

```diff
@@ -18,36 +18,76 @@
 
 namespace Surfnet\AzureMfa\Domain;
 
+use Exception;
 use Surfnet\AzureMfa\Domain\Exception\InvalidUserIdException;
 
 class UserId
 {
+    const SEPARATOR = '|';
+    const VALID_UNIQUE_ID = '/^[a-z0-9]{6}-[a-z0-9]{4}$/';
```

Comment:
why are there two parts to the VALID_UNIQUE_ID separated by a `-` ?

Reply:
Because it was shamelessly stolen from Tiqr and I didn't want to reinvent the wheel:
https://github.com/OpenConext/Stepup-tiqr/blob/da33decfecb6ce4bf6a8e2bdf2c396b0130be945/src/AppBundle/Tiqr/Legacy/TiqrService.php#L198